### PR TITLE
Add tests for recurring event functionality

### PR DESCRIPTION
**Pull request by codeGobbler**
@pjsier @peterntiamoah please review.. I've added specs to test the schedule and calendar_events methods for the Event model. #18

### CODE REVIEW

**Comment by pjsier** [comment]
@codeGobbler quick heads up, you can make tracking reviews easier by adding me and Peter in the Reviewers column on the right. That adds some functionality around approvals or comments than mentioning us

**Comment by codeGobbler** [comment]
Thanks @pjsier for that info, still learning new things every day lol

File: spec/models/event_spec.rb [hunk 1]
```diff
@@ -3,10 +3,7 @@
 require 'rails_helper'
 
 RSpec.describe Event, type: :model do
-  subject do
-    Event.new(description: 'Sunday Service', start_time: '2021-07-17', end_time: '2021-09-17',
-              event_location: '234 Gebroni Road, Raleigh, NC 12345')
-  end
+  subject { FactoryBot.create(:event) }
```
Comment (pjsier) [inline spec/models/event_spec.rb:6]: Comment 1

File: spec/models/event_spec.rb [hunk 2]
```diff
@@ -22,4 +19,30 @@
...
+    end
+    it 'should have recurrence rules' do
+      expect(subject.schedule(:all_recurrence_rules)).to_not be_nil
+    end
+  end
+  context 'can calendar events' do
+    it 'has a calendar_events method that returns an array' do
+      expect(subject.calendar_events(Time.now)).to be_an_instance_of(Array)
+    end
+    it 'returns an array with expected values' do
```
Comment (pjsier) [inline spec/models/event_spec.rb:43]: Comment 2

**Review by pjsier — CHANGES_REQUESTED** [review]
Requesting changes

File: spec/models/event_spec.rb [hunk 3]
```diff
@@ -22,4 +19,30 @@
...
+    it 'should have a start time' do
+      expect(subject.schedule(:start_time)).to_not be_nil
+    end
+    it 'should have an end time' do
+      expect(subject.schedule(:end_time)).to_not be_nil
+    end
+    it 'should have recurrence rules' do
+      expect(subject.schedule(:all_recurrence_rules)).to_not be_nil
+    end
+  end
```
Comment (pjsier) [inline spec/models/event_spec.rb:38]: Line block

**Comment by peterntiamoah** [comment]
Ice_Cube testing has been merged.

**Review by peterntiamoah — COMMENTED** [review]
Changes have been reviewed. Everything looks good. Keep up the excellent work Alton

**Pull request merged**